Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Temp, not sure what to do here yet");
+        }
+        
+        // Takes the weight and height and returns BMI
+        public static float BMICalculation(float weight, float height)
+        {
+            return weight / (height * height);
+        }
+
+        // Prompts the user for an input, converts it to an integer and returns it
+        public static int IntInput(string message)
```

Comment:
PromptInt is the name I would expect. Why? Because you ask for inputting an integer and such name tell that. IntInput is a bit technical, not obvious, up for interpretation and thus is not as good.